controller/src/main/java/io/pravega/controller/server/SegmentStoreConnectionManager.java

```diff
@@ -298,6 +298,11 @@ public void conditionalCheckFailed(WireCommands.ConditionalCheckFailed dataNotAp
             execute(ReplyProcessor::conditionalCheckFailed, dataNotAppended);
         }
 
+        @Override
+        public void storageFlush(WireCommands.StorageFlush storageFlush) {
```

Comment:
Why do we need this method in the Controller? Can we get rid of it?

Reply:
`storageFlushed` is a method in the `Reply` interface. It needs to be implemented here as the class implements `Reply`.